collector/lib/CollectorConfig.h

```diff
@@ -41,10 +42,9 @@ class CollectorArgs;
 class CollectorConfig {
  public:
   static constexpr bool kUseChiselCache = true;
-  static constexpr bool kSnapLen = 0;
   static constexpr bool kTurnOffScrape = false;
   static constexpr int kScrapeInterval = 30;
-  static constexpr char kCollectionMethod[] = "kernel-module";
+  static constexpr collectionMethod kCollectionMethod = KERNEL_MODULE;
```

Comment:
Since we're defaulting to EBPF these days, we should probably set this to follow that default

Reply:
Yeah, we probably should, but this PR is already quite tainted by changes that are not specific to adding the `CO.RE eBPF` candidate, mind if I do it in a follow up?